### PR TITLE
Synth Grid Paste Bug-fix

### PR DESCRIPTION
**Pull request by cupcake99**
Hey,

The synth grid was allowing a paste when no synth had been copied (eg. after program launch) leading to a reproduce-able crash every time. I added a little guard routine to the pasteSynth() method to prevent this.

Just thought i'd open a pull request to this issue i found and fixed, as a test to make sure i'm following the right code-upload-pull procedure before i get going pushing the other changes as individual branches.

Sorry i've been slow on this, am not on internet much at the moment.

Ok, all the best!

### CODE REVIEW

**Review by cupcake99 — COMMENTED** [review]
The refreshView() method is called because wires were not being redrawn properly if a Container was in the pasted synth. For some reason calling it fixes that problem. Maybe something you could look into. I couldn't figure it out.

**Comment by kometbomb** [comment]
It might be that the wires don't get rebuilt because a Container has to figure out the number of inputs/outputs so in the first pass it might not know that number and it skips the connected wires.

This is a very good find and a good PR. :+1:

File: src/SynthGrid.cpp [hunk 1]
```diff
@@ -938,16 +938,21 @@ void SynthGrid::copySynth()
...
+	{
+		ModularSynth& synth = getModularSynth();
+		synth.lock();
+		synth.copy(*mCopyBuffer);
+		synth.unlock();
+	
 	rebuildWires();
-
+	
+	refreshView();
```
Comment (kometbomb) [inline src/SynthGrid.cpp:953]: Can you please indent this properly before merge? Alternatively, you can just do 
```
if (mCopyBuffer == NULL)
{
  return;
}
```

very first in the method and leave the rest untouched.

**Comment by cupcake99** [comment]
Ok, fixed it.

Your method worked very nicely. Thanks.

**Comment by kometbomb** [comment]
No, thank you for the bugfix. :)

**Pull request merged**